### PR TITLE
Solution to  #285: drag-test demo

### PR DESCRIPTION
**Pull request by josrr**
Please review.

### CODE REVIEW

File: Core/clim-core/frames.lisp [hunk 1]
```diff
@@ -1630,10 +1630,12 @@ have a `pointer-documentation-pane' as pointer documentation,
 		   do (return-from find-dest-translator translator))
 	     nil)
 	   (do-feedback (window x y state)
-	     (funcall feedback-fn frame from-presentation window
-			initial-x initial-y x y state))
+	     (when (typep window 'clim-stream-pane)
```
Comment (dkochmanski) [inline Core/clim-core/frames.lisp]: it solves immediate issue for me, but are we allowed to limit d&d to only clim-stream-pane? Also, why does it break, if window is something different than clim-stream-pane?

File: Core/clim-core/frames.lisp [hunk 2]
```diff
@@ -1630,10 +1630,12 @@ have a `pointer-documentation-pane' as pointer documentation,
...
+	       (funcall feedback-fn frame from-presentation window
+			initial-x initial-y x y state)))
 	   (do-hilite (presentation window state)
-	     (funcall hilite-fn frame presentation window state))
+	     (when (typep window 'clim-stream-pane)
```
Comment (dkochmanski) [inline Core/clim-core/frames.lisp]: is this also necessary? after wrapping in when feedback-fn, things seem to work for me. why is the same thing done to do-hilite?

Comment (josrr) [reply Core/clim-core/frames.lisp]: You are right, s not necessary; I didn't test this after changed button-release by pointer-button-release.

File: Core/clim-core/frames.lisp (hunk 1 above)
Comment (josrr) [reply Core/clim-core/frames.lisp]: I think is not specified; the problem is that the function frame-drag-and-drop-feedback try to draw in the stream, and this invoke INVOKE-WITH-SHEET-MEDIUM-BOUND; but there is no method defined for all classes of sheets.

Comment (dkochmanski) [reply Core/clim-core/frames.lisp]: in that case please specialize `frame-drag-and-drop-feedback` stream argument on `output-stream class ` and add new method (above the existing one), which is not specialized on anything and is a no-op (empty body).

We'd want to specialize on `output-recording-stream`, but we have take into account also that we have `encapsulating-stream`, which isn't recording (but underlaying stream may be output-recording-stream).

Comment (dkochmanski) [reply Core/clim-core/frames.lisp]: also please squash changes into single commit when you do

**Comment by dkochmanski** [comment]
also, before you squash, please remove all indent and whitespace changes from changes

**Comment by dkochmanski** [comment]
hey, I've squashed it and pushed to the repository. please claim a bounty on bountysource. Thanks for narrowing down and fixing the issue.

**Pull request closed without merging**